SWLOR.Game.Server/Service/CraftService/RecipeType.cs

```diff
@@ -716,7 +716,7 @@ public enum RecipeType
         WildBloodBroth = 2078,
         GrandioseSandwich = 2079,
         WildStew = 2080,
-        HssissSurprise = 2081,
+        KlorslugSurprise = 2081,
```

Comment:
This change is going to break. Database entities track recipes created by enumeration name and removing HssissSurprise is going to cause exceptions to be raised upon serialization/deserialization.

Simplest thing is to change this back. Less simple is to create a new recipe. Most complex is to do a data migration.